sirepo/util.py

```diff
@@ -30,11 +30,23 @@
 AUTH_HEADER = "Authorization"
 
 #: http auth header scheme bearer
-AUTH_HEADER_SCHEME_BEARER = "Bearer"
+_AUTH_HEADER_SCHEME_BEARER = "Bearer"
```

Comment:
I feel like this should be somewhere else. It's very specific to http so maybe http_util.py.

Reply:
Is it worth creating http_util.py just for this? It is only used in this file. Maybe move it and the uses to http_util.py?